htdocs/fichinter/list.php

```diff
@@ -222,6 +226,15 @@
 if (is_array($extrafields->attribute_label) && count($extrafields->attribute_label)) $sql.= " LEFT JOIN ".MAIN_DB_PREFIX."fichinter_extrafields as ef on (f.rowid = ef.fk_object)";
 if (empty($conf->global->FICHINTER_DISABLE_DETAILS) && $atleastonefieldinlines) $sql.= " LEFT JOIN ".MAIN_DB_PREFIX."fichinterdet as fd ON fd.fk_fichinter = f.rowid";
 if (! $user->rights->societe->client->voir && empty($socid)) $sql.= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
+if(!empty($conf->global->FICHINTER_LIST_SHOW_INVOICES)) {
```

Comment:
The relation fichinter - invoice is a n-n relation like most relation object-object defined by llx_element_element table.
Adding such a left join will generate a lof of duplicate record.
I recommand to NOT change the $sql and later, into the 
if (! empty($arrayfields['inter.facref']['checked']))
{
            print "<td>"
...Make a loop to get all invoices linked to intervention...
           print "</td>"
}